### PR TITLE
Cookies Implementation

### PR DESCRIPTION
**Pull request by itsaliu90**
-Creates and drops cookie with unique sessionId for each browser that visits site for the first time.
-This sessionId will be used to generate or retrieve a unique cart for each user from the DB, _even if they are anonymous_ (not authenticated)
-VIDEO!!! Om nom nom nom

cookies

### CODE REVIEW

File: browser/js/app.js [hunk 1]
```diff
@@ -8,6 +8,19 @@ app.config(function ($urlRouterProvider, $locationProvider) {
     $urlRouterProvider.otherwise('/');
 });
 
+// Create cookie controller
+
+app.controller('CookieCtrl', ['$scope','$cookieStore', function($scope, $cookieStore) {
+  // Setting a cookie to store the sessionId
+  var sessionId = Math.random().toString(36).slice(2);
```
Comment (carolynyoo) [inline browser/js/app.js:15]: whats the base 36 rationale?

File: browser/js/app.js [hunk 2]
```diff
@@ -8,6 +8,19 @@ app.config(function ($urlRouterProvider, $locationProvider) {
     $urlRouterProvider.otherwise('/');
 });
 
+// Create cookie controller
+
+app.controller('CookieCtrl', ['$scope','$cookieStore', function($scope, $cookieStore) {
```
Comment (carolynyoo) [inline browser/js/app.js:13]: is $cookieStore deprecated? https://docs.angularjs.org/api/ngCookies/service/$cookieStore

**Comment by carolynyoo** [comment]
I :heart: COOKIES!
ok to merge after comments are taken into account

File: browser/js/app.js (hunk 2 above)
Comment (itsaliu90) [inline browser/js/app.js:13]: Yeah apparently, but I couldn't get it to work with `$cookies` and all the docs I could find were with `cookieStore`

File: browser/js/app.js (hunk 1 above)
Comment (itsaliu90) [inline browser/js/app.js:15]: It allows for the inclusion of letters (10 numbers + 26 letters), see: http://code.tutsplus.com/tutorials/understanding-base-36-math--cms-21677.

Comment (carolynyoo) [inline browser/js/app.js:15]: :+1:

File: browser/js/app.js (hunk 2 above)
Comment (carolynyoo) [inline browser/js/app.js:13]: ok yolo :+1:

**Pull request merged**